resource/reactivedrop_english.txt

```diff
@@ -3535,6 +3535,100 @@
 
 "asw_boss_title_muon_gem"	"Muon Gem"
 
+// Reduction strings
+"red1_1_title"	"Go to the Docking Office"
+
+"red1_2_title"	"Go to Station Control"
+
+"red1_3_title"	"Get to the Escape Pod Room and Evacuate"
+
+"red1_gametext_1"	"(Enable multiplayer game instructor to see dialogue and hints)"
```

Comment:
@jhh8 I don't think there is a differentiation between single- and multiplayer game instructor, so that word could be removed